clang/runtime/dpct-rt/include/dpct/shmem_utils.hpp

```diff
@@ -0,0 +1,45 @@
+//==--------------- shmem_utils.hpp ---------------*- C++ -*----------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __DPCT_SHMEM_UTILS_HPP__
+#define __DPCT_SHMEM_UTILS_HPP__
+
+namespace dpct::shmem {
+
+/// Setup Operations
+/// Initialize Intel SHMEM runtime based on exisiting communicator in attributes
+/// \param [in] runtime_flags specifies the exisiting communicator
```

Comment:
```suggestion
/// \param [in] runtime_flags specifies the exisiting communicator
```
could you be more detail on the runtime_flags?

Reply:
Done